docs/specviz/displaying.rst

```diff
@@ -24,4 +24,26 @@ More words...
 Plot Settings
 =============
 
-More words...
\ No newline at end of file
+To access plot settings for a particular viewer (including the spectrum viewer), click the "hammer and screwdriver" icon, followed by the "gear" icon, followed by the "Layer" tab.
+
+.. image:: img/specviz_plot_settings.png
+
+Layer
+-----
+
+The top section of the "Layer" tab contains options to change the color of the spectrum (click the "square" icon to see a color change menu), change visibility of the spectrum ("eye" icon), and a drop-down box to select which layer will have its settings changed.
+
+Line Width
+----------
+
+Width of the spectrum in pixels. Larger values correspond to thicker lines on the plot.
+
+Vmin and Vmax
+-------------
+
+Minimum and maximum values of the y axis.
+
+Percentile
+----------
+
+Sets the bounds of the plot (Vmin and Vmax) such that the selected percentage of the data is shown in the viewer. Editing either bound manually changes the "Percentile" selection to "Custom."
```

Comment:
I noticed that there is an :guilabel:`up/down' button at the bottom of the Layer tab that will renormalize the spectrum.  I was unaware of it until yesterday.  Do you want to document what it does here?

Reply:
Actually, it is in the Viewer tab, not the layer tab...my bad.